lib/banner.dart

```diff
@@ -55,7 +57,13 @@ class DFPBanner extends StatelessWidget {
         onAdOpened: onAdOpened,
         onAdClosed: onAdClosed,
         onAdLeftApplication: onAdLeftApplication,
-        onPlatformCompleted: (_DFPBannerViewController controller) => controller.load(),
+        onAdViewCreated: onAdViewCreated,
+        onPlatformCompleted: (DFPBannerViewController controller) {
+          controller.load();
```

Comment:
I want to distinguish the role of the method.

Here we call the _init() method

```
controller._init();
```


At the time of reload call the ```reload()``` method.

Here is a new signature:

```
  Future<void> _init() async {
    _channel.setMethodCallHandler((call) {
      switch (call.method) {
        case "onAdLoaded":
          onAdLoaded();
          break;
        case "onAdFailedToLoad":
          var map = call.arguments.cast<String, int>();
          onAdFailedToLoad(map['errorCode']);
          break;
        case "onAdOpened":
          onAdOpened();
          break;
        case "onAdClosed":
          onAdClosed();
          break;
        case "onAdLeftApplication":
          onAdLeftApplication();
          break;
      }
    });
    return _load();
  }

  Future<void> reload() async {
    return _load();
  }

  Future<void> _load() {
    return _channel.invokeMethod('load', {
      "isDevelop": this.isDevelop,
      "testDevices": this.testDevices?.values,
      "adUnitId": this.adUnitId,
      "adSizes": [this.adSize.toString()],
      "widths": [this.adSize.width],
      "heights": [this.adSize.height],
    });
  }
```

Reply:
Done